### ISSUE TITLE
Add a setting to switch the font to a specialised one for low-vision people

### ISSUE THREAD

**Issue by blabaere** [elementary/settings-desktop#390]
### Problem

A given font's legibility depends also on the person reading it, and while there is a setting to change the size of the text there is currently no way to choose a font targeted at low-vision people.

### Proposal

Add a toggle in "System Settings / Desktop / Text" that would change the fonts to specialized ones.
For example, this would replace Inter with one of the fonts available at the [Braille Institute](https://brailleinstitute.org/freefont).
In Code and Terminal,  [Intel One Mono](https://github.com/intel/intel-one-mono) could be used.

### Prior Art (Optional)

This is almost a clone of the existing "Dyslexia-friendly" toggle.
![Settings_Desktop_Text](https://github.com/elementary/switchboard-plug-pantheon-shell/assets/9447137/824f3c06-871f-47ce-a9ee-8f057992ddd0)

**Comment by teamcons** [comment]
toggling one while the other is active would untoggle the active one, correct ?

**Comment by blabaere** [comment]
Yes, I think it would make sense to be able to select only one of  "Dislexia-friendly" or "Low-vision", if I understand the question correctly.
